RedHat-UX/red-hat-design-system
# docs: new footer docs

## What I did

1. [x] Implement new information architecture (IA) for element pages
2. [x] Add a new top level menu item "Elements"
3. [x] Redirect `/components/footer/*` to `/elements/footer/*`
4. [x] Do same for demos

## Testing Instructions

1. Check out the [DP](https://deploy-preview-638--red-hat-design-system.netlify.app/elements/footer/)
2. [Basic demo](https://deploy-preview-638--red-hat-design-system.netlify.app/elements/footer/demo/) should work
3. [Nested demo](https://deploy-preview-638--red-hat-design-system.netlify.app/elements/footer/demo/global-footer/) should work
4. [Accordion demo](https://deploy-preview-638--red-hat-design-system.netlify.app/components/accordion/demo/) should work
3. [Accordion Nested demo](https://deploy-preview-638--red-hat-design-system.netlify.app/components/accordion/demo/color-context/) should work
5. [/components/footer/](https://deploy-preview-638--red-hat-design-system.netlify.app/components/footer/) should redirect to /elements/footer/
6. [/components/footer/demo/](https://deploy-preview-638--red-hat-design-system.netlify.app/components/footer/demo/) should redirect to /elements/footer/demo/ and work
7. [/components/accordion/](https://deploy-preview-638--red-hat-design-system.netlify.app/components/accordion/) should not redirect (for a subsequent PR)
3. [/components/accordion/demo/](https://deploy-preview-638--red-hat-design-system.netlify.app/components/accordion/demo/) should not redirect (for a subsequent PR)
## Notes to Reviewers
**NB**: :point_right: This PR begins a process of moving all the content in `/components/` to either `/elements/` or `/patterns/` :point_left: 

Element pages are now located at `/elements/${tagName}/` instead of `/components/${tagName}/`. This is in anticipation of `/patterns/${patternName}` TBD. As well, content for element pages is moved from `/docs/components/footer.njk` to `/elements/rh-footer/docs/*.md`
